redcap_bridge/utils.py

```diff
@@ -122,3 +122,44 @@ def remove_columns(csv_file, compressed_file=None):
 def exportCSVtoXLS(csv_file, compressed_file=None):
     read_file = pd.read_csv(csv_file, na_filter=False, dtype='str')
     read_file.to_excel(r'Path', index=None, header=True)
+
+
+def conversion_csv_to_json(csv_file):
+    """
+    Test conversion function
+    """
+    df = pd.read_csv(csv_file)
+    json_df = df.to_dict()
+    elab_json = {}
+    elab_dict = {}
+
+    list_of_dict = df.to_dict('records')
+    for redcap_field_dict in list_of_dict:
+        if redcap_field_dict['Field Type'] == 'text':
+            if redcap_field_dict['Text Validation Type OR Show Slider Number'] == 'number' or redcap_field_dict['Text Validation Type OR Show Slider Number'] == 'integer':
+                elab_dict = text_to_json(redcap_field_dict)
```

Comment:
in this case you are only dealing with numbers, so I propose to also make this clear in the name of the conversion function. And at this level we are returning a dictionary, so `_to_dict` might be a better naming, too.
```suggestion
                elab_dict = number_to_dict(redcap_field_dict)
```